charts/linkerd2-multicluster/templates/service-mirror.yaml

```diff
@@ -1,3 +1,4 @@
+{{if .Values.serviceMirror -}}
 ---
 kind: Namespace
 apiVersion: v1
```

Comment:
The namespace yaml in service-mirror.yaml can be removed now that `namespace.yaml` is being included